eppo/src/main/java/cloud/eppo/android/dto/Range.java

```diff
@@ -1,6 +1,6 @@
 package cloud.eppo.android.dto;
 
-public class ShardRange {
+public class Range {
```

Comment:
Why was this renamed? It's still the returned type by `createShardRange` so it's still a shard range? Should that function be renamed too for consistency or does `Range` served other purposes as well?